java/ingest/ingest-runner/src/main/java/sleeper/ingest/IngestFactory.java

```diff
@@ -146,6 +143,7 @@ private static Configuration defaultHadoopConfiguration() {
         Configuration conf = new Configuration();
         conf.set("fs.s3a.aws.credentials.provider", "com.amazonaws.auth.EC2ContainerCredentialsProviderWrapper");
         conf.set("fs.s3a.fast.upload", "true");
+        conf.set("fs.s3a.bucket.probe", "false");
```

Comment:
Looking at the Hadoop code, this needs to be an integer:

https://github.com/apache/hadoop/blob/rel/release-3.3.3/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java#L655

Reply:
It also looks like the bucket check is turned off by default in the version we're using:

https://github.com/apache/hadoop/blob/rel/release-3.3.3/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java#L558

Reply:
Good shout, I don't think this issue is needed then since it's turned off by default

Reply:
Actually looking at the switch case, there's another default where if it's set to an unrecognised value it checks for the bucket existence against v2 of the S3 API. It might be best to set it explicitly.